cilium-dbg/cmd/endpoint_get.go

```diff
@@ -50,26 +47,10 @@ var endpointGetCmd = &cobra.Command{
 			endpointInst = append(endpointInst, result)
 		}
 
-		if command.OutputOption() {
```

Comment:
Why do we need to remove plaintext output?  Isn't this still useful for debugging to have the human readable format?